cv32e40p_manifest.flist

```diff
@@ -58,3 +58,4 @@ ${DESIGN_RTL_DIR}/cv32e40p_sleep_unit.sv
 ${DESIGN_RTL_DIR}/cv32e40p_core.sv
 ${DESIGN_RTL_DIR}/../bhv/cv32e40p_sim_clock_gate.sv
 ${DESIGN_RTL_DIR}/../bhv/cv32e40p_tracer.sv
```

Comment:
The other bhv files will need to be put in this list as well now (apu tracer and log)

Reply:
I force pushed this change as well